pkg/controllers/vdb/podfacts.go

```diff
@@ -431,7 +431,7 @@ func (p *PodFacts) genGatherScript(vdb *vapi.VerticaDB, pf *PodFact) string {
 		echo -n 'vnodeName: '
 		cd %s/%s/v_%s_node????_catalog 2> /dev/null && basename $(pwd) | rev | cut -c9- | rev || echo ""
 		echo -n 'verticaPIDRunning: '
-		[[ $(pgrep ^vertica) ]] && echo true || echo false
+		[[ $(pgrep -f "^.*vertica\s-D") ]] && echo true || echo false
```

Comment:
Why is this change needed?

Reply:
Because the server container container starts there is already a vertica process polling for the config file, so the node was always reported up. This is to search for `/opt/vertica/bin/vertica -D***` which is the actual start command. This is just a work around to allow me to progress.

Reply:
This brings up an interesting case. How are we able to determine if vertica process is ready for connections now. We can't do this by checking the process. I guess check for -D works as long as the vertica process runs as a daemon. Maybe we should check the HTTPS endpoint? We can call /v1/version. If that returns HTTP code 200, then vertica is up.

Reply:
That works too and is consistent with the health probe. What I did here is just expanding the regex to match `/opt/vertica/bin/vertica -D /data/vertdb/v_vertdb_node0001_catalog***`(-D stands for db-path) but not `/opt/vertica/bin/vertica --stattup-config ***`

Reply:
I think we should check endpoint in this PR. Otherwise, it will be something we need to fix when we change the vertica process to stop running as a daemon.

Reply:
I agree, I will make the change

Reply:
Now I kept this because in `checkIfNodeIsDoingStartup`  when `verticaPIDRunning` is false, we return immediately. Without this, as soon as the pod is running i`verticaPIDRunning`  but that's just the start cmd polling the file and I do not want to set `verticaPIDRunning` based on that. Maybe we can keep it until we fix the daemon jira.

Reply:
Okay. Leave a comment with the jira then if you are going to address this later. I had another comment about using verticaPIDRunning less. It would be nice if it was used only for admintools deployments.

Reply:
We can. There is only one place we use it. I just need to figure out to not use it for vclusterops